recipes/breakpad/all/conanfile.py

```diff
@@ -31,7 +31,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("linux-syscall-support/cci.20200813")
+        self.requires("linux-syscall-support/cci.20200813", transitive_headers=True)
```

Comment:
Thanks for your patience @artem-ogre, I'm adding this change to take the chance to make it v2 compatible. I think it's worth it.